bot/cogs/grades_data/grades_cog.py

```diff
@@ -266,23 +250,28 @@ async def grades(self, ctx, course, year=2014):
         '''
         
         try:
-            embed = discord.Embed(title="Grades", color=Colors.ClemsonOrange)
-            result = self.go(course, year)
-            embed.add_field(name="Result", value=result, inline=False)
-            prefix = await self.bot.get_prefix(ctx)
-            if isinstance(prefix, list):
-                prefix = prefix[0]
-            exp = f'Type `{prefix}help grades` for more information' # NOQA
-            embed.add_field(name='Explanation', value=exp)
+            if str(year) in self.year_list:
+                embed = discord.Embed(title="Grades", color=Colors.ClemsonOrange)
+                result = self.go(course, year)
+                embed.add_field(name="Result", value=result, inline=False)
+                prefix = await self.bot.get_prefix(ctx)
+                if isinstance(prefix, list):
+                    prefix = prefix[0]
+                exp = f'Type `{prefix}help grades` for more information' # NOQA
+                embed.add_field(name='Explanation', value=exp)
+            else:
+                embed = discord.Embed(title="Grades", color=Colors.Error)
+                result = f'Year not available. Reminder that years go from {self.year_list[0]}-{self.year_list[len(self.year_list)-1]}'
```

Comment:
`{self.year_list[0]}-{self.year_list[len(self.year_list)-1]}` can just be 

`{self.year_list[0]}-{self.year_list[-1]}`